src/measures/localMeasures/LocalMeasure.cpp

```diff
@@ -4,9 +4,10 @@
 
 using namespace std;
 
-const string LocalMeasure::autogenMatricesFolder = "matrices/autogenerated/";
+const string LocalMeasure::autogenMatricesFolder = "autogenerated/matrices/";
 
 LocalMeasure::LocalMeasure(Graph* G1, Graph* G2, string name) : Measure(G1, G2, name) {
+    createFolder(autogenMatricesFolder);

```

Comment:
Somewhere, I think in each and every localMeasure, you also need to create the subdirectory of the name of the measure.